src/main/frontend/db/model.cljs

```diff
@@ -379,7 +379,7 @@
                                                               (= k (:db/id (:block/left block))))
                                                             blocks)
                                                     (map #(select-keys % [:db/id :block/level :block/content :block/file])))}))))]
-         (util/pprint duplicates)))
+         #_(util/pprint duplicates)))
```

Comment:
This and `with-out-str` was the cause of the bug. I disabled this for now but we should be logging instead of having any print statements. I can create a story to address all print statements